src/lib/datepicker/calendar-footer.html

```diff
@@ -0,0 +1,9 @@
+<div class="mat-calendar-footer cdk-visually-hidden">
```

Comment:
@jelbourn shouldn't we show the button if it receives focus? Otherwise people that are tabbing through the dialog without screen readers will land on it with no feedback. See how GitHub does it with their "Skip to content" button.

![github_-_google_chrome_2018-12-09_09-57-00](https://user-images.githubusercontent.com/4450522/49695317-95682a80-fb99-11e8-9ddf-32e492ed01a7.png)